.NET/Microsoft.Recognizers.Text.Sequence/Japanese/JapaneseURLExtractorConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions;
+using Microsoft.Recognizers.Definitions.Chinese;
+
+namespace Microsoft.Recognizers.Text.Sequence.Japanese
```

Comment:
Why create a Japanese config and extractor? We don't need these as there is not Japanese-specific change. It's just a CN/JP vs. EN fork, like the previous PR, no?
Keep the Japanese tests though.